### ISSUE TITLE
Continuous build and test

### ISSUE THREAD

**Issue by laughedelic** [inkytonik/atom-sbt#8]
Would be nice to be able to run commands like `~compile` and get errors in the linter cleared every time it's rerun (now if you run `~compile` same errors get accumulated)

**Comment by inkytonik** [comment]
Good suggestion. I don't use the continuous commands so I hadn't thought of this problem. Will look into it.

**Comment by inkytonik** [comment]
e9f9df6 provides a solution for this one. Will be in 0.7.0. When that version is available please let me know here if it's working ok for you.

**Comment by laughedelic** [comment]
Wonderful! 
I used [SublimeSBT](https://github.com/jarhart/SublimeSBT) plugin for a long time, but then I switched to Atom and there were no such plugin for it. Finally, I get the same (and even better) experience in Atom! Thank you! :+1: 

One thing that I noticed with this update: if the `~compile` is running and you call, for example `Sbt: Exit` command, it doesn't work as expected:

```
...
10. Waiting for source changes... (press enter to interrupt)
exit
>
```

I guess, it should insert additional newline (or a space) before the command.
